### PR TITLE
Fix bunch of scripts

### PR DESCRIPTION
**Pull request by kmsiapps**
updated spadenadefix.py
fixed `/where` (on movements.py)
fixed nointelonwalls.py

### CODE REVIEW

File: piqueserver/scripts/spadenadefix.py [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+version 2(2017.12.23)
 """
 
+from pyspades.constants import SPADE_TOOL
 
 def apply_script(protocol, connection, config):
     class SpadenadeConnection(connection):
-
+        def on_secondary_fire_set(self, secondary):
+            self.wasSpade = (self.tool == SPADE_TOOL)
```
Comment (NotAFile) [inline piqueserver/scripts/spadenadefix.py]: while you are at it make this `was_spade` according to pep8 perhaps

**Comment by kmsiapps** [comment]
Fixes #218

File: piqueserver/core_commands/movement.py [hunk 2]
```diff
@@ -77,7 +77,7 @@ def where(connection, player=None):
         raise ValueError()
     x, y, z = connection.get_location()
     return '%s is in %s (%s, %s, %s)' % (
-        connection.name, to_coordinates(x, y), int(x), int(y), int(z))
+        connection.name, to_coordinates(int(x), int(y)), int(x), int(y), int(z))
```
Comment (NotAFile) [inline piqueserver/core_commands/movement.py]: Perhaps it would be better to have to_coordinates be able to deal with floats?

Comment (kmsiapps) [reply piqueserver/core_commands/movement.py]: @NotAFile But it fixes the problem o_o Just tested it again.

**Pull request merged**